.github/workflows/daily-build.yml

```
@@ -12,3 +12,4 @@ jobs:
     secrets: inherit
     with:
       repo-name: module-ballerinax-redis
+      additional-build-flags: "-PbuildUsingDocker=\"\""
```

We can just use the property without any value.

```suggestion
      additional-build-flags: "-P buildUsingDocker"
```